mobile/src/bundles/auth/components/sign-in-form/sign-in-form.tsx

```diff
@@ -51,7 +59,7 @@ const SignInForm: React.FC<Properties> = ({ onSubmit }) => {
             <View style={styles.formWrapper}>
                 <FormField
                     errorMessage={errors.email?.message}
-                    label="Email"
+                    label="Email*"
```

Comment:
If we really need this * in the label (because in the design we don't have this) just pass to FormField `required`